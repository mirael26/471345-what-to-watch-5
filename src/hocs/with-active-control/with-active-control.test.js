import React from "react";
import renderer from "react-test-renderer";

import withActiveControl from "./with-active-control";

const MockComponent = () => {
  return (
    <div>
    </div>
  );
};

const MockComponentWrapped = withActiveControl(MockComponent);

it(`Should withActiveControl render correctly`, () => {
  const noop = () => {};
  const tree = renderer
    .create(
        <MockComponentWrapped
          isPlaying={false}
          onPlayButtonClick={noop}
          isFullscreen={false}
          onFullScreenButtonClick={noop}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
