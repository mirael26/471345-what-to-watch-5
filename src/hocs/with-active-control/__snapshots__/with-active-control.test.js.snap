// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should withActiveControl render correctly 1`] = `<div />`;
